app/src/components/common/icons/IconSettings.tsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react'
+import styled from 'styled-components'
 
-import theme from '../../../theme'
-
+const Path = styled.path`
+  fill: ${props => props.theme.colors.textColorLighter};
+`
```

Comment:
I don't think this is the best practice to target path like this. Instead it would be better to target whole svg and then target property path inside it and then fill property within it. Like so
const Svg=styled.svg`
	path{
		fill: ${props => props.theme.colors.textColorLighter};
	}
`